src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,66 @@
+package parser;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.testng.annotations.Test;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.testng.Assert.assertNotNull;
```

Comment:
Remove testng annotations and use junit only

Reply:
done